EIPS/eip-2228.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 2228
+title: Canonicalize the name of network with chain ID 1
+author: William Entriken (@fulldecent)
+discussions-to: https://github.com/ethereum/EIPs/issues/2228
+status: Draft
+type: Informational
+created: 2019-08-04
+---
+
+## Simple Summary
+
+The Ethereum network with chainid 1 is referenced from authoritative sources using several names that are conflicting. This standard specifies the name for this network and provides reference examples in an effort to standardize the word choice and provide a common language for use to use to refer to this network.
```

Comment:
This summary is too long.  Thin email subject title or GitHub pull request title.  Perhaps something like:
> Defines the naming convention of Ethereum Mainnet.

Reply:
Also, this simple summary currently includes motivational content, which should be moved to the Motivation section.